conf/fluo-deploy.props.example

```diff
@@ -69,6 +71,12 @@ key_name = my_aws_key
 # Tags to add instances
 #instance_tags = key1:value1,key2:value2
 
+[baremetal]
+mount_root = /var/disk
+mounts = /var/disk01,/var/disk02,/var/disk03
+mount_devices = /dev/hdb1,/dev/hdc1,/dev/hdd1
+network_interface=eth0
```

Comment:
network_interface needs to be set when running in EC2 so this should probably be moved to general